config/locales/en.yml

```diff
@@ -0,0 +1,56 @@
+en:
+  label_redmine_zulip: Zulip
+  zulip_settings_header: Zulip Plugin Configuration
+  zulip_settings_label_url: Zulip API URL
+  zulip_settings_label_email: Zulip Bot Email
+  zulip_settings_label_api_key: Zulip Bot API key
+  field_zulip_stream: Stream name
+  field_zulip_private_messages: Get private notifications
+  field_zulip_subject_issue: Create subjects for issues
+  field_zulip_subject_version: Create subjects for versions
+  zulip_notify_assignment: >
+    %{user} **assigned** issue [%{id}: %{subject}](%{url}) to **you**
```

Comment:
I would recommend ending this part with a colon (`:`). Currently, I am doing an audit of our entire integrations code base to make sure that we use proper punctuation wherever possible. :)